cadquery/occ_impl/geom.py

```diff
@@ -366,15 +366,15 @@ def __repr__(self) -> str:
         """
         Generate a valid python expression representing this Matrix
         """
-        list = [ [self.__getitem__((r,c)) for c in range(4)] for r in range(4)]
-        return "Matrix("+"".join(str(list)).replace("],","],\n       ")+")"
+        matrix_str = ",\n        ".join(str(self.transposed_list()[i::4]) for i in range(4))
```

Comment:
You are calling transposed_list() in 4 times. Each time you throw away most of the result it returns.
I think it should be called once and result should be stored in temporary variable for later use.
Same with __str__(..) below

Reply:
I went to far in an attempt for clarity .. the code only calls `transpose_list` once now. By the way, Adam didn't want the `__str__` method so I removed it. Thanks for your suggestions!